.gitignore

```diff
@@ -11,6 +11,7 @@ contracts/leverage_trading/target
 /node_modules/
 /res/
 /.near-credentials
+contracts/leverage_trading/res/*.wasm
```

Comment:
change it contracts/leverage_trading/res/*.wasm => contracts/leverage_trading/res/

Reply:
Changed it to `contracts/leverage_trading/res/*`. Because the `contracts/leverage_trading/res/` ignores the folder